modules/ezcontent_block/modules/ezcontent_block_media/ezcontent_block_media.info.yml

```diff
@@ -0,0 +1,7 @@
+name: Media Block
+type: module
+description: "Provides Media Block Types Based Features."
+package: EZContent Block
+core: 8.x
```

Comment:
Let's make this module D9 compatible by defining `core_version_requirement: ^8.8 || ^9`